pages/develop.md

```diff
@@ -216,6 +218,27 @@ In that location, you will find your Firefox profile directories, e.g. 12a3bc4d.
 1. You should see a dialog asking to confirm the Privly extension installation. 
 1. Success! If it didn't work, see the tip from above, where the filename in your development profile folder may need to be privly@priv.ly or {ec8030f7-c20a-464f-9b0e-13a3a9e97384}.
 
+#### Jetpack Extension ####
+
+**This extension is not yet packaged.**
+
+To develop this add-on, you will need:
+* Firefox version >= 38.0
+* jpm
+
+For jpm installation, follow the instructions as mentioned [here](https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/jpm#Installation).
+Firefox v38.0 and greater can be found [here](http://ftp.mozilla.org/pub/firefox/releases/).
+
+1) Get the [source code](https://github.com/privly/privly-jetpack) from GitHub. You can download it by cloning the repository,
+`git clone --recursive https://github.com/privly/privly-jetpack.git`.
+2) Build [Privly Applications](https://priv.ly/pages/develop.html#PrivlyApplications)
```

Comment:
Should be `[Privly Applications](#PrivlyApplications)` since it is an anchor link.